Makefile

```diff
@@ -24,6 +24,10 @@ GO ?= go
 GOFLAGS := 
 # Set to 1 to use static linking for all builds (including tests).
 STATIC := $(STATIC)
+# The cockroach image to be used for starting Docker containers
+# during acceptance tests. Usually cockroachdb/cockroach{,-dev}
+# depending on the context.
+COCKROACH_IMAGE :=
```

Comment:
Could also prefix this variable with `export` in order to put it into the environment. Then you could remove the `export COCKROACH_IMAGE` line below.